documentation/RLC-quickstart.md

```diff
@@ -105,7 +105,7 @@ In order to release it, we need to add some tests for it to make sure we are del
 
 ## Prerequisites
 
-- To record and playback the tests, [Docker](https://www.docker.com/) is required when we run the test, as the [test proxy server](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy) is run in a container during testing. When running the tests, ensure the Docker daemon is running and you have permission to use it. For WSL 2, running `sudo service docker start` and `sudo usermod -aG docker $USER` should be sufficient.
+- To record and playback the tests, [Docker](https://www.docker.com/) is required when we run the test, as the [test proxy server](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy) is run in a container during testing. When running the tests, ensure the Docker daemon is running and you have permission to use it.
```

Comment:
Can we add the link of test-quickstart.md in this doc?

Reply:
I plan to merge them first and then add the absolute href link in the quickstart